### DISCUSSION TITLE
Translate to Slovenian language?

### DISCUSSION THREAD

**Discussion by thehijacker** [crocodilestick/Calibre-Web-Automated#77] [Q&A] [answered]
Hi,

Where can I start to have the GUI translated to Slovenian language. Is there a Crowdin or Weblate project for it somewhere?

Thanks.

**Comment by iroQuai** [comment, marked as answer]
Localization isn't part of the calibre-web-automated project. It's part of the base repo Calibre-Web. More info on contributing translations can be found here:

https://github.com/janeczku/calibre-web/blob/master/CONTRIBUTING.md#translation

**Reply by thehijacker** [reply]
I will make a PR with Slovenian translation in base project then. Can I send the .po file also to you or you will pull them from base project?

**Reply by crocodilestick** [reply]
> I will make a PR with Slovenian translation in base project then. Can I send the .po file also to you or you will pull them from base project?

If you notify me here when it's done I can add it to the next version of CWA

**Reply by thehijacker** [reply]
I have added the Slovenian translation two days ago. Yesterday I checked it with Calibre Web nightly build and made additional changes to it. There is PR to apply them, but was not yet merged. Here is the PR:

https://github.com/janeczku/calibre-web/pull/3154

Here is the latest message.po file for Slovenian translation (folder in Calibre Web - https://github.com/janeczku/calibre-web/tree/master/cps/translations/sl/LC_MESSAGES)

[messages.zip](https://github.com/user-attachments/files/17070552/messages.zip)

Are there any additional strings for Calibre Web Automated that needs to be translated? Where can I find them?

**Reply by thehijacker** [reply]
Updated Slovenian translation wes merged to Calibre-Web. Can you please pull them for the next release so the GUI will include the Slovenian translation? Thank you.
